tests/components/wunderground/test_sensor.py

```diff
@@ -114,7 +119,7 @@ async def test_sensor(hass, aioclient_mock):
     assert state.state == "40"
     assert state.name == "Feels Like"
     assert "entity_picture" not in state.attributes
-    assert state.attributes["unit_of_measurement"] == TEMP_CELSIUS
+    assert state.attributes[ATTR_UNIT_OF_MEASUREMENT] == TEMP_CELSIUS
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)